flixel/graphics/frames/FlxAtlasFrames.hx

```diff
@@ -53,39 +53,56 @@ class FlxAtlasFrames extends FlxFramesCollection
 		
 		var data:Dynamic = Json.parse(Description);
 		
-		var rotated:Bool;
-		var name:String;
-		var sourceSize:FlxPoint;
-		var offset:FlxPoint;
-		var angle:FlxFrameAngle;
-		var frameRect:FlxRect;
-		
-		for (frame in Lambda.array(data.frames))
+		// JSON-Array
+		if (Std.is(data.frames, Array))
 		{
-			rotated = frame.rotated;
-			name = frame.filename;
-			sourceSize = FlxPoint.get(frame.sourceSize.w, frame.sourceSize.h);
-			offset = FlxPoint.get(frame.spriteSourceSize.x, frame.spriteSourceSize.y);
-			angle = FlxFrameAngle.ANGLE_0;
-			frameRect = null;
-			
-			if (rotated)
+			for (frame in Lambda.array(data.frames))
 			{
-				frameRect = new FlxRect(frame.frame.x, frame.frame.y, frame.frame.h, frame.frame.w);
-				angle = FlxFrameAngle.ANGLE_NEG_90;
+				texturePackerHelper(frame.filename, frame, frames);
 			}
-			else
+		}
+		
+		// JSON-Hash
+		else
+		{
+			for (frameName in Reflect.fields(data.frames))
```

Comment:
On second thought, I guess we're already working with Dynamic anyway... (which isn't ideal to begin with).